scripts/sell_unused_slots.py

```diff
@@ -126,13 +130,28 @@ def _lambda_invocation_counts(self, function_names: Iterable[str]) -> Dict[str,
 
 
 def main(argv):
+    parser = argparse.ArgumentParser(description='Delete BigQuery reservation '
+                                                 'resources if no ongoing '
+                                                 'reindex is detected.',
```

Comment:
String block looks a bit odd but I think there is no way around it.

Reply:
I think I found an example that shows a better way.

Reply:
Nicely done!